spec/TestSpec.ts

```diff
@@ -0,0 +1,23 @@
+import { getTextNodeAccessibleName } from '../src/accessible-name';
+import { JSDOM } from 'jsdom';
```

Comment:
I've done a bit more research into JSDOM and I don't think it's what we want to use for these tests. Various DOM APIs are implemented in pure JavaScript which is running in a browser in these tests. But that means that we're not actually using the browser's implementation so the test isn't really running in Chrome.

The good news is that if you remove lines 6 and 7 it _might_ just work?

Reply:
I think this did work when I tried it in the past, and it seems to use the browser DOM API through Karma. I'll double check this to make sure.